test/functional/test_runner.py

```diff
@@ -161,6 +161,7 @@
     'rpc_bind.py --ipv6',
     'rpc_bind.py --nonloopback',
     'mining_basic.py',
+    'mining_timewarp_fork.py',
```

Comment:
The test takes a long time, so better move it up to approx where `mining_getblocktemplate_longpoll` is